src/Controller/OrderController.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace App\Controller;
+
+
+
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Validation;
+use App\Entity\Order;
+use App\Entity\OrderState;
+use App\Entity\OrderVehicleData;
+use App\Entity\VehicleMake;
+use App\Entity\VehicleModel;
+use App\Entity\Category;
+use App\Entity\User;
+
+class OrderController extends AbstractController
+{
+
+
+    /**
+     * @Route("/order", name="order")
+     * @param Request $request
+     * @return Response
+     */
+    public function submit(Request $request)
+    {
+
+        $message = "order failed";
+        $validator = Validation::createValidator();
+
+        //$content=trim(file_get_contents("php://input"));
+        //$input = json_decode($content, true);
+
+        $input = json_decode($request->getContent(),true);
```

Comment:
https://symfony.com/doc/current/components/http_foundation.html#accessing-request-data
pasiskaityk kaip gaut submitintus duomenis is requesto.